librz/io/p/io_shm.c

```diff
@@ -35,7 +42,11 @@ static int shm__write(RzIO *io, RzIODesc *fd, const ut8 *buf, int count) {
 		(void)memcpy(shm->buf + io->off, buf, count);
 		return count;
 	}
+#if !defined(__WINDOWS__)
 	return write(shm->fd, buf, count);
+#else
+	return 0;
+#endif
```

Comment:
Isn't it possible to write/read on Windows?

Reply:
It uses the path of `if (shm->buf) ...` But maybe some comment could be added.

Reply:
@ret2libc you cant read/write to the mapped file handle, the code will always use the memory mapped address instead.